src/app/icd/client/DefaultICDClientStorage.h

```diff
@@ -156,7 +159,8 @@ class DefaultICDClientStorage : public ICDClientStorage
     {
         // All the fields added together
         return TLV::EstimateStructOverhead(sizeof(NodeId), sizeof(FabricIndex), sizeof(uint32_t), sizeof(uint32_t),
```

Comment:
Might be worth while ading /**/ comments for what each field represents.

Reply:
updated, thanks